src/databox/azext_databox/tests/latest/test_databox_scenario.py

```diff
@@ -0,0 +1,95 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer, JMESPathCheck)
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class DataBoxScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_databox')
+    @StorageAccountPreparer()
+    def test_databox(self):
+        job_name = self.create_random_name('job', 24)
+        storage_account = self.cmd('storage account show -n {sa}').get_output_in_json()
+        storage_account_id = storage_account['id']
+        self.kwargs.update({
+            'job_name': job_name,
+            'storage_account_id': storage_account_id
+        })
+
+        self.cmd('databox job create '
+                 '--resource-group {rg} '
+                 '--job-name {job_name} '
+                 '--location westus '
+                 '--sku-name DataBox '
+                 '--contact-name "Public SDK Test" '
+                 '--phone 14258828080 '
+                 '--email-list testing@microsoft.com '
+                 '--street-address1 "1 MICROSOFT WAY" '
+                 '--city Redmond '
+                 '--state-or-province WA '
+                 '--country US '
+                 '--postal-code 98052 '
+                 '--company-name Microsoft '
+                 '--storage-account-id {storage_account_id} ',
+                 checks=[JMESPathCheck('status', 'DeviceOrdered')])
+
+        self.cmd('databox job update '
+                 '--resource-group {rg} '
+                 '--job-name {job_name} '
+                 '--contact-name "Public SDK Test 1" '
+                 '--email-list testing1@microsoft.com ',
+                 checks=[])
```

Comment:
can we add some checks here? just like the show command

Reply:
the updated parameters are not returned from command and I checked the update result by the following 'show' command

---
In reply to: [388072600](https://github.com/Azure/azure-cli-extensions/pull/1344#discussion_r388072600) [](ancestors = 388072600)

Reply:
it's quite strange. Is it a long running operation? It should return something.

Reply:
Yes, it returns something, but the response doesn't include updated properties. "Show" command below should cover the update logic.

---
In reply to: [390126507](https://github.com/Azure/azure-cli-extensions/pull/1344#discussion_r390126507) [](ancestors = 390126507)